src/redux/categories/categories.js

```diff
@@ -0,0 +1,18 @@
+const CHECK_STATUS = 'CHECK_STATUS';
```

Comment:
- [ ] As already mentioned above, you would need to scope your action types in regards to the feature they represent. Let's fix this issue, please.